chia/cmds/configure.py

```diff
@@ -71,13 +72,39 @@ def configure(
         else:
             print("uPnP disabled")
         change_made = True
+    if testnet is not None:
```

Comment:
Question: is there another config variable that might conflict with this one, e.g. a "mainnet" variable?